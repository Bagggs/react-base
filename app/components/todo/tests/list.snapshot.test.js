import React from 'react';
import renderer from 'react-test-renderer';
import TodoList from '../list';

describe('TodoList', () => {
  it('renders correctly', () => {
    const todos = [
      { id: 1, isComplete: false, name: 'Something to do 1' },
      { id: 2, isComplete: false, name: 'Something to do 2' },
      { id: 3, isComplete: false, name: 'Something to do 3' }
    ];
    const todoListComponent = renderer.create(<TodoList todos={ todos } />);

    expect(todoListComponent.toJSON()).toMatchSnapshot();
  });
});
