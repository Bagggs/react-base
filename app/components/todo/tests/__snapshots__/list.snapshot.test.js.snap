// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList renders correctly 1`] = `
<div
  className="list-group"
>
  <button
    className="todo list-group-item"
    onClick={[Function]}
    type="button"
  >
    <span>
      Something to do 1
    </span>
    <span
      className="glyphicon glyphicon-trash iconTrash"
      onClick={[Function]}
    />
  </button>
  <button
    className="todo list-group-item"
    onClick={[Function]}
    type="button"
  >
    <span>
      Something to do 2
    </span>
    <span
      className="glyphicon glyphicon-trash iconTrash"
      onClick={[Function]}
    />
  </button>
  <button
    className="todo list-group-item"
    onClick={[Function]}
    type="button"
  >
    <span>
      Something to do 3
    </span>
    <span
      className="glyphicon glyphicon-trash iconTrash"
      onClick={[Function]}
    />
  </button>
</div>
`;
